playwright/integrations/unauthenticated-chat-with-masking.spec.ts

```diff
@@ -0,0 +1,54 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import { test, expect } from '@playwright/test';
+import ACSEndpoints from '../utils/ACSEndpoints';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('UnauthenticatedChat');
+
+test.describe('UnauthenticatedChat @UnauthenticatedChatWithMasking', () => {
+    test('ChatSDK.sendMessage() should send the message masked', async ({ page }) => {
+        await page.goto(testPage);
+
+        const content = "4111111111111111";
+
+        const [getMessagesRequest, getMessagesResponse, runtimeContext] = await Promise.all([
```

Comment:
We should rely on send message request than get message request. 

We're trying to validate sending message and not getting message. 

You can use the test case `ChatSDK.sendMessage() should send a message with default tags & proper metadata` as reference

Reply:
Modified by referring the above test case.